distsql/select_result.go

```diff
@@ -81,6 +81,13 @@ func (r *selectResult) Fetch(ctx context.Context) {
 }
 
 func (r *selectResult) fetchResp(ctx context.Context) error {
+	startTime := time.Now()
+	defer func() {
+		duration := time.Since(startTime)
```

Comment:
Sorry for the roundtrip, but I think it's better to just gauge the time of `r.resp.Next` and exclude the deserialization time, same as previous PR #13274

Reply:
You are right, we should not just gauge the running time here, since it is not that crucial.

Reply:
I've updated the code, PTALA @SunRunAway